pages/glossary/visual-section-of-content.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Visual section of content
+key: visual-section-of-content
+unambiguous: true
+objective: false
+---
+
+Something in the line of "a subtree rooted at an element with distinguishing style" using https://github.com/act-rules/act-rules.github.io/blob/896b28caeb240171d3b2db626506a7c6034ce20b/pages/glossary/distinguishing-styles.md
+
++note that these only make sense for sighted users and users of AT need to rely on something else.
+
+Problem: the def linked above is about distinguishing between two elements. This def should be about the element being distinguishable from the surrounding.
```

Comment:
I don't think distinguishing style works for this. What I would suggest is something like this:

```md
Section of the content

A section of the content is a group of elements, visually presented together, usually to show
they have a common purpose. A number of characteristics should be considered when deciding if
two elements are in the same section of content:

1. **boundaries**: elements separated by borders, box shadow or by having different backgrounds
are often not part of the same section of the content

2. **layout**: elements in a different column or row are often not part of the same section of
the content

3. **whitespace and negative space**: elements separated by spacing that is greater than the
default spacing between paragraphs are often not part of the same section of content

#### Subsections

Sections are often nested, creating a visual hierarchy in the page. In addition to the first
three indicators, the following presentational mechanisms are often used to indicate which
elements belong in the same subsection of the content:

1. **Headings**: If the first element in a section is presented as a heading, the section only
includes one such heading. The next element that is presented as a heading of that same style,
or of a style where the heading has a larger font size, or the same font size and a greater
font weight is the start of another section.

2. **Text styling**: Elements with a distinct text style are often a subsection of the content.
Text styles include such things as color, font, size, weight, use of shadow or outline, etc. 
Text presented as a heading is not a subsection of content.

3. **Indentation**: Elements with a greater indentation are often a subsection of the content.

4. **Separators**: Decorative non-text content is often used to indictate that a section can be 
divided into subsections. Elements in the same section, separated by this decorative non-text 
content are often not part of the same subsection. Separators often take the form of a line or 
simple graphic.
```